build_scripts/appimage/AppRun

```diff
@@ -24,7 +29,19 @@ do
     fi
 done
 
+
 export PYTHONNOUSERSITE=1
-(PYTHONHOME="${APPDIR}/opt/python3.11" exec "${APPDIR}/opt/python3.11/bin/python3.11" "${APPDIR}/usr/src/monkey_island.py" $@)
+
+export run_island="${APPDIR}/opt/python3.11/bin/python3.11 ${APPDIR}/usr/src/monkey_island.py"
+
+if [ "$1" == "service_run" ]; then
+    (PYTHONHOME="${APPDIR}/opt/python3.11" ${run_island} "$@")
```

Comment:
It would be cleaner to use an environment variable instead of a parameter. You can add an environment variable to the systemd unit lIke `Environment=ISLAND_SERVICE=1`

Reply:
that and passing a parameter in systemd unit is not trivial